src/app/components/carousel/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import classnames from 'classnames';
+import ReactSwipe from 'react-swipe';
+
+const Carousel  = React.createClass({
```

Comment:
Why do you use `React.createClass` in this case but `class` with the rest of the stateful components? Unless there is a strong reason I'd stick to a single convention.